examples/example_info.py

```diff
@@ -0,0 +1,65 @@
+"""Example on how to read mask version and properties from a KNX actor."""
+import asyncio
+import sys
+from typing import List
+
+from xknx import XKNX
+from xknx.core import PayloadReader
+from xknx.telegram import IndividualAddress
+from xknx.telegram.apci import (
+    DeviceDescriptorRead,
+    DeviceDescriptorResponse,
+    PropertyValueRead,
+    PropertyValueResponse,
+)
+
+
+async def main(argv: List[str]):
+    """Connect and read information from a KNX device. Requires a System B device."""
+    if len(argv) == 2:
+        address = IndividualAddress(argv[1])
+    else:
+        address = "1.1.1"
+
+    xknx = XKNX()
+    await xknx.start()
+
+    reader = PayloadReader(xknx, address)
+
+    # Read the mask version of the device (descriptor 0).
+    payload = await reader.send(
+        DeviceDescriptorRead(descriptor=0), response_class=DeviceDescriptorResponse
+    )
+    if payload is not None:
+        print("Mask version: %04x" % payload.value)
+
+    # Read the serial number of the device (object 0, property 11).
+    payload = await reader.send(
+        PropertyValueRead(object_index=0, property_id=11, count=1, start_index=1),
+        response_class=PropertyValueResponse,
+    )
+    if payload is not None:
+        print(
+            "Serial number: {:02x}{:02x}:{:02x}{:02x}{:02x}{:02x}".format(
+                payload.data[0],
+                payload.data[1],
+                payload.data[2],
+                payload.data[3],
+                payload.data[4],
+                payload.data[5],
+            )
+        )
+
+    # Check if the device is in programming mode (object 0, property 54).
+    payload = await reader.send(
+        PropertyValueRead(object_index=0, property_id=54, count=1, start_index=1),
+        response_class=PropertyValueResponse,
```

Comment:
So would we now be able to respond correctly to an ETS programming mode scan? 
Or even a Line scan (I think this would also require point-to-point connection-oriented link to be supported ?)

Reply:
We would now be able to parse (and generate) ETS programming telegrams. But sending them may require a point-to-point connection-oriented link in some cases.

The new APCI services would be the building blocks to do so. In my [own](https://github.com/basilfx/RIOT/tree/feature/knx) implementation/actor (that I can program using ETS), I use them as well.

My specific use-case: my actors have over-the-wire update support. I (abuse) MemoryWrite and PropertyValueRead/Write to do so. With this PR, I can create a small application that can flash updates. I don't use a point-to-point connection-oriented link in this case.

Reply:
> We would now be able to parse (and generate) ETS programming telegrams.

Does this mean, we could capture the binaries and write them to a file?

Reply:
Yes, as long as the telegrams are captured, it is possible.

Reply:
cool. I'm really curious to see, how the binary changes, if you e.g. modify a group address ...

(One day, i would like to program my devices without having to boot ETS :-)